### DISCUSSION TITLE
Request failed with status code 500 - when i click on dashboard, Alerts and hunt

### DISCUSSION THREAD

**Discussion by Nazeer1982** [Security-Onion-Solutions/securityonion#11605] [2.4]
### Version

2.4.2

### Installation Method

Network installation on Ubuntu

### Description

installation

### Installation Type

Distributed

### Location

cloud

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

128

### Storage for /

800

### Storage for /nsm

600

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I setup server on a 20.04 cloud instance Server manager search node

Worked fine till i  tried to add a hardware sensor

/root/sosetup.log: line 2: -----------------------------: command not found
/root/sosetup.log: line 3: Detecting: command not found
/root/sosetup.log: line 4: -----------------------------: command not found
/root/sosetup.log: line 6: 2023-10-24T11:31:51Z: command not found
/root/sosetup.log: line 6: Running: command not found
/root/sosetup.log: line 6: INFO: command not found
/root/sosetup.log: line 7: Executing: command not found
/root/sosetup.log: line 8: syntax error near unexpected token `('
/root/sosetup.log: line 8: `W: https://packages.microsoft.com/ubuntu/22.04/prod/dists/jammy/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.'
azuadmin@masteronion:/opt/so/log$

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
Have you looked at https://docs.securityonion.net/en/2.4/os.html? Ubuntu is one of the unsupported OS. 

Have you looked at using the SO cloud image?
